src/api.ts

```diff
@@ -0,0 +1,49 @@
+const apiURL: string = process.env.REACT_APP_EXPLORER_API_URL || "";
```

Comment:
Is this a duplicate from an old merge? Please merge development back into your branch.